calico_node/glide.lock

```diff
@@ -150,6 +159,15 @@ imports:
   version: ca53cad383cad2479bbba7f7a1a05797ec1386e4
 - name: github.com/peterbourgon/diskv
   version: 5f041e8faa004a95c88a202771f4cc3e991971e6
+- name: github.com/projectcalico/calico
```

Comment:
Do we really want to be vendoring the calico repo *from* the calico repo?  I think that'll mean that we'll have to commit, push, update vendor and then commit and push again.

Reply:
Nope we don't want to, once the clients package is moved into libcalico-go that can go away.

Reply:
Is someone already making that move to libcalico-go?

Reply:
Apparently I already did because it was inside the package I was using